src/components/PathwaysList/PathwaysListCheckbox.tsx

```diff
@@ -0,0 +1,62 @@
+import { ChangeEvent, useCallback, useMemo, useState } from 'react';
```

Comment:
Since this is a hook and not a component, and since we will want to reuse it for the criteria list, why don't we create a `src/hooks` folder to keep it in. We could also move the `useRefState` hook from utils into this new folder as well.